Python/Product/Cookiecutter/View/CookiecutterSearchPage.xaml

```diff
@@ -80,8 +85,27 @@
             <!-- Description -->
             <RowDefinition Height="Auto"/>
         </Grid.RowDefinitions>
+
+        <!-- Created successfully -->
+        <StackPanel Grid.Row="0" Orientation="Vertical" Visibility="{Binding Path=IsCreatingSuccess, Converter={StaticResource BoolToVisibleOrCollapsed}}">
+            <StackPanel Margin="4 4 4 8"
+                        Orientation="Horizontal">
+                <imaging:CrispImage Width="16" Height="16" Moniker="{x:Static imagecatalog:KnownMonikers.StatusOK}"/>
+                <TextBlock Margin="4 0 0 0" TextWrapping="Wrap" Text="Files created successfully."/>
+            </StackPanel>
+            <StackPanel Margin="4 4 4 8"
+                        Orientation="Horizontal">
+                <imaging:CrispImage Margin="20 0 0 0" Width="16" Height="16" Moniker="{x:Static imagecatalog:KnownMonikers.OpenFolder}"/>
+                <Button Style="{StaticResource NavigationButton}"
+                    Margin="4 0 0 0"
+                    Command="{x:Static vm:CookiecutterViewModel.OpenInExplorer}"
+                    CommandParameter="{Binding Path=OpenInExplorerFolderPath,Mode=OneWay}"
+                    HorizontalAlignment="Left">Open in solution explorer</Button>
```

Comment:
Solution Explorer should be capitalized. I also prefer to put text on a new line (leading and trailing whitespace is trimmed unless you add a special attribute to prevent it).